docs/release-notes/8.3.asciidoc

```diff
@@ -5,6 +5,11 @@
 [[release-notes-8.3.3]]
 === 8.3.3
 
+[discrete]
+[[known-issue-8.3.3]]
+==== Known issues
+* An {endpoint-sec} bug on macOS and Linux can cause CPU spikes if Malware prevention is enabled on an {endpoint-cloud-sec} integration policy. When this happens, {endpoint-sec} is locked up and stops functioning normally. To avoid this issue, we recommend using {stack} 8.3.2 or earlier. If you are using 8.3.3 and have encountered this issue, you can temporarily resolve it by rebooting your computer and disabling Malware prevention on your {endpoint-cloud-sec} integration policy (https://github.com/elastic/endpoint/issues/22[#22]).
```

Comment:
```suggestion
* An {endpoint-sec} bug on macOS and Linux can cause CPU spikes if malware prevention is enabled on an {endpoint-cloud-sec} integration policy. When this happens, {endpoint-sec} locks up and has performance issues. To avoid this issue, we recommend using {stack} version 8.3.2 or earlier. If you are using {stack} version 8.3.3 and have encountered this issue, you can temporarily resolve it by rebooting your computer and disabling malware prevention on your {endpoint-cloud-sec} integration policy (https://github.com/elastic/endpoint/issues/22[#22]).
```

Reply:
Thanks for these edits, @jmikell821 ! I think I might need to change the {stack} attribute to {agent} as per @ferullo 's comment [here](https://github.com/elastic/security-docs/pull/2292/files#r943725501).

Reply:
@jmikell821 making some minor edits to your comment (switching `{stack}` with `{agent}` so I can commit your changes :)

Reply:
@nastasha-solomon @ferullo sounds like we should also change `malware prevention` to just maybe `malware protection`?

Reply:
Changing to `malware protection` sounds good.

Reply:
> may have performance issues

I think this edit should be changed to something like "may not fully protect the system". Because of the state Endpoint is in when this happens some or all Endpoint features may stop working right.

Reply:
👍 sounds great